packages/drink-buddy-ui/prisma/schema.prisma

```diff
@@ -10,7 +10,57 @@ datasource db {
   url      = "postgresql://postgres:Admin0@localhost:5432/postgers" //TODO: env("DATABASE_URL")
 }
 
+model User {
+  id                           Int      @id @default(autoincrement())
```

Comment:
I believe that we should use String ID with given length to unify ID system in our app - later on it will come in handy when we will generate links for user profile, we cant just go like `.../app/user?userId=1`